create-custom-map-style/README.md

```diff
@@ -0,0 +1,990 @@
+# Amazon Location Service - Create a custom map style
```

Comment:
[markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) is awesome for ensuring consistency across Markdown files (other READMEs in this repo use it to stay clean).

Reply:
done and pushed